.github/workflows/codeql-analysis.yml

```diff
@@ -18,8 +18,8 @@ on:
       - "!src/electionguard-ui/**"
       - "test/**"
     branches: [main]
-  schedule:
-    - cron: "0 7 * * 4"
+  # schedule:
```

Comment:
why are we running this on a schedule?

Reply:
We moved it to a schedule since the task takes 4-6 HOURS to complete and wanted to get this away from the normal PR process.